walkthroughs/eks/o11y-prometheus.md

```diff
@@ -8,7 +8,32 @@ Prometheus is a systems and service monitoring system. It collects metrics from
 
 ### Option 1: Quick setup
  
-App Mesh provides a basic installation to setup Prometheus quickly using Helm. To install the Prometheus pre-configured to work with App Mesh, follow the instructions in [appmesh-prometheus](https://github.com/aws/eks-charts/blob/master/stable/appmesh-prometheus/README.md) Helm charts.
+App Mesh provides a basic installation to setup Prometheus quickly using Helm. To install the Prometheus pre-configured to work with App Mesh:
+1. Enable EBS CSI Driver:  
```

Comment:
Are these instructions needed ot use the appmesh-prometheus chart? If so it seems like we should add them there rather than pulling them into this repo.

Reply:
Yes these are appmesh-prometheus changes. I originally thought we didn't have eks charts change access, let me move these over